src/System.Management.Automation/utils/tracing/PSEtwLog.cs

```diff
@@ -118,6 +118,16 @@ internal static void LogProviderLifecycleEvent(LogContext logContext, string pro
             provider.LogProviderLifecycleEvent(logContext, providerName, newState);
         }
 
+        /// <summary>
+        /// Provider interface function for logging Amsi init event.
+        /// </summary>
+        /// <param name="InitFailed"></param>
+        /// <param name="context"></param>
+        internal static void LogAmsiInitEvent(string InitFailed, string context)
```

Comment:
It looks like this event is only for when Amsi initialization fails, so it should be named:
```csharp
internal static void LogAmsiInitFailedEvent(string message, string context)
```